### PR TITLE
Add Prettier formatting to Nextra site

### PR DESCRIPTION
**Pull request by blakezimmerman**
### Description

I added formatting and lint scripts to the Nextra app so that its files are formatted/checked when the repo's `pnpm fmt` and `pnpm lint` scripts are run.

### Checklist

- Do all Lints pass?
  - [x] Have you ran `pnpm spellcheck`?
  - [x] Have you ran `pnpm fmt`?
  - [x] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/nextra/package.json [hunk 1]
```diff
@@ -12,7 +12,9 @@
     "prebuild": "node nextra-remote-filepaths/fetch.js",
     "predev": "pnpm prebuild",
     "start": "next start -p 3030",
-    "types:check": "tsc --noEmit"
+    "types:check": "tsc --noEmit",
+    "fmt": "prettier --write '**/*.(tsx|ts|jsx|js|json|md|html)'",
+    "lint": "prettier --check '**/*.(tsx|ts|jsx|js|json|md|html)'"
```
Comment (hariria) [inline apps/nextra/package.json:17]: Should this be `mdx`?

**Review by hariria — APPROVED** [review]
Noice haha, though I just realized CI doesn't check linting or formatting atm 🤡 . Maybe will address it in the future.

File: apps/nextra/package.json (hunk 1 above)
Comment (blakezimmerman) [reply apps/nextra/package.json:17]: Ah yeah, I should add `mdx` there. I just ripped this straight from the docusaurus package.json

Comment (blakezimmerman) [reply apps/nextra/package.json:17]: I just ran the format script on all of our `mdx` files and it made some strange formatting changes that broke a bunch of the files. I guess Prettier support for MDX isn't there yet and we'll have to go without it for now.

**Pull request merged**